src/luminol/utils.py

```diff
@@ -42,7 +43,8 @@ def read_csv(csv_name):
     :return dict: a dictionary represents the data in file.
     """
     data = {}
-    if not isinstance(csv_name, (str, unicode)):
+    # if not isinstance(csv_name, (str, unicode)):
```

Comment:
In this case, if you don't care whether `csv_name` is unicode or bytes, you could do something like this:

```
import sys

if int(sys.version[0]) == 2:
    str_types = (str, unicode)
else:
    str_types = (bytes, str)
```

Then `if not isinstance(csv_name, str_types):` ...

Reply:
I like this, this  is a cleaner approach.